docs/package-management.md

```diff
@@ -267,3 +267,119 @@ The advantage of this approach —as opposed to vendoring the JavaScript package
 inside the bindings— is that it gives users of the bindings complete flexibility
 over the way these JavaScript packages are downloaded and bundled.
 
+## Finding and using Melange compatible packages
+
+### opam packages
+
+Melange packages are usually available on [opam](https://opam.ocaml.org). Package search is available in the opam CLI via `opam search <package-name>`, e.g. `opam search reason-react`. You can run `opam install <package-name>` to download, build and install opam packages in your switch. Remember that opam won't automatically add the dependency to `<your-project>.opam` file, so it must be added manually:
```

Comment:
> Melange packages are usually available on opam.

This is a bit of a stretch 😬  as I think only 1 Melange package is available in the official opam repository today (`reason-react`). But that's the ultimate goal, and we have to be optimistic 🚀  that over time more packages will be published, so I would leave it as is.